README.md

```diff
@@ -1,24 +1,272 @@
-# Requirements
+Corva python-sdk is a framework for building stream, scheduled and task apps.
+
+## Contents
+
+- [Requirements](#requirements)
+- [Installation](#installation)
+- [App types](#app-types)
+- [Event](#event)
+- [Api](#api)
+- [Cache](#cache)
+- [Contributing](#contributing)
+
+## Requirements
+
 Python 3.8+
 
-# Local development
+- [Pydantic](https://github.com/samuelcolvin/pydantic)
+- [Redis](https://pypi.org/project/redis/)
+- [Requests](https://pypi.org/project/requests/)
+- [urllib3](https://pypi.org/project/urllib3/)
+
+## Installation
+
+```console
+$ pip install corva-sdk
+```
+
+## App types
+
+There are three app types, that you can build: `stream`, `scheduled` and `task`.<br>
+**Note**: it is recommended to use type hints like in examples below,
+so that editors and tools can give you better support.
```

Comment:
This recommendation does not change the way how our library works. Let's believe that users choose using or not using type hints by themselves

Reply:
Ok